packages/playground/src/views/solutions_view.vue

```diff
@@ -1,12 +1,6 @@
 <template>
   <view-layout>
-    <v-autocomplete
-      label="Search Applications"
-      v-model="searchItem"
-      :items="titles"
-      class="mb-5"
-      @update:search="filterCards"
-    ></v-autocomplete>
+    <v-text-field label="Search Applications" v-model="searchItem" class="mb-5" @input="filterCards"></v-text-field>
```

Comment:
I think it should be clearable

Reply:
what do you mean?

Reply:
clearable prop in vuetify `v-text-field`

Reply:
Added.